doc/mgr/influx.rst

```diff
@@ -0,0 +1,192 @@
+=============
+Influx Plugin 
+=============
+
+The influx plugin continuously collects and sends time series data to an influxdb database. 
+Users have the option to specify what type of stats they want to collect. Some default counters are already set. 
+However, users will have the option to choose some additional counters to collect 
+
+-------------
+Configuration 
+-------------
+
+In order for this module to work, a configure file must be created ``/etc/ceph/influx.conf``
```

Comment:
The other mgr modules have moved to keeping their config on the mon config-key database.  Is there any reason we can't do the same here?  From teh module you can just get_config() to fetch values.